### PR TITLE
Adding binary tree python code

### PR DESCRIPTION
**Pull request by shatmanyu**
## Have you read the Contribution Guidelines?

(Write your answer here.)

## Description

(Write your answer here.)

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue 
<br/>
(Write your answer above.)

### CODE REVIEW

**Comment by shatmanyu** [comment]
Hey there!
I have came across your repository, it's great for preparation of Ds-Algos.
I have added a binary tree python code containing all basic fundamentals of bt including -> Traversals,Mirror,height,Diameter and min-max of bt.

**Comment by itslinotlie** [comment]
Welcome to this repo! You may have missed some finer details in how workflow is processed in this repo, but it's okay, I'll guide you through what I know so far.

- Generally, the workflow starts from the Issues. This is where you either solve an existing issue, or create a new one yourself (in your case, you would either contribute to an existing Binary Tree issue i.e. #229 or create a new issue containing everything).
- The important thing before you start coding is to wait to be **assigned** the issue before working on the code (which afaik, you missed)
- Then you can code everything up, and remember to add your changes to the respective README file (which you also have not done).

tldr; it might be beneifical for you to re-read the README and contributing guidelines.

(just my 2 cents though, I'm just a guy on the internet)

**Comment by sunilgknair051** [comment]
@itslinotlie , I would mostly close this request.
But, I will that decision for you now.

**Comment by itslinotlie** [comment]
Closed for not meeting the repo requirements. Thanks @sunilgknair051 for the guidance
